packages/services/hmi-server/src/main/resources/application.properties

```diff
@@ -23,6 +23,7 @@ mp.openapi.extensions.smallrye.info.description=REST endpoints for the TERArium
 ########################################################################################################################
 %dev.data-service/mp-rest/url=http://localhost:3020
 %prod.data-service/mp-rest/url=http://${data-service}:3020
+model-service/mp-rest/url=http://localhost:3010
```

Comment:
you need to set a %dev and a %prod version of the model service. The prod version will expect an environment parameter of ${model-service}.

You will also need to update the deployment scripts.

This will now be a breaking change PR

Reply:
I don't know what deployment scripts you are talking about - can you point me to a relevant PR that does similar thing. Or better yet - maybe a quick walkthrough with the team because this will probably come up again.

Reply:
the deployment scripts in the `orchestration` repo. The ones that start the Kubernetes cluster for our local development and for staging

Reply:
I updated the scripts already

Reply:
done